src/coprocessor/dag/rpn_expr/impl_math.rs

```diff
@@ -25,7 +25,7 @@ fn abs_uint(arg: &Option<Int>) -> Result<Option<Int>> {
 #[rpn_fn]
 #[inline]
 fn abs_real(arg: &Option<Real>) -> Result<Option<Real>> {
-    // abs returns NAN if the number is NAN, so don't worry about it
+    // arg is never nan
```

Comment:
So why not simply remove this comment 🤣 This comment provides no information, which looks to be useless.